src/registry_schemas/example_data/schema_data.py

```diff
@@ -2599,6 +2610,7 @@
     ('registrarsNotation', REGISTRARS_NOTATION),
     ('registrarsOrder', REGISTRARS_ORDER),
     ('consentContinuationOut', CONSENT_CONTINUATION_OUT),
+    ('continuationOut', CONTINUATION_OUT),
```

Comment:
Will there be a non-corp continuation out that will require a new name -- should this one be corpContinuationOut or continuationOutCorp? Or is that something to be handled separately?

Will this be like corrections?

Reply:
Probably not